website/pages/docs/advanced-topics/monitoring/otel-grafana.md

```diff
@@ -0,0 +1,253 @@
+
+## Step 1: Creating a Docker Compose file
+
+We will use [Tempo][Tempo] for ingesting traces, [Loki][Loki] for logs, [Prometheus][Prometheus] for metrics, and the [OpenTelemetry collector][Collector] for collecting and forwarding the data to each service.
+
+Create a file named `docker-compose.yml` with the following content:
+
+```yaml
+version: "3.8"
+services:
+  tempo:
+    image: grafana/tempo:latest
+    command: ["-config.file=/etc/tempo.yaml"]
+    volumes:
+      - tempo_data:/tmp
+      - ./tempo/tempo.yaml:/etc/tempo.yaml
+    ports:
+      - "3200"
+      - "4318"
+  loki:
+    image: grafana/loki:latest
+    ports:
+      - "3100:3100"
+    command: -config.file=/etc/loki/local-config.yaml
+  collector:
+    image: otel/opentelemetry-collector-contrib:latest
+    ports:
+      - "4318:4318"
+      - "8090"
+    volumes:
+      - ./collector/collector.yaml:/etc/otelcol-contrib/config.yaml
+  prometheus:
+    image: prom/prometheus:latest
+    command:
+      - "--enable-feature=remote-write-receiver"
+      - "--config.file=/etc/prometheus/prometheus.yaml"
+    ports:
+      - "9090:9090"
+    volumes:
+      - prometheus:/prometheus
+      - ./prometheus/prometheus.yaml:/etc/prometheus/prometheus.yaml
+  grafana:
+    image: grafana/grafana-enterprise
+    volumes:
+      - grafana_data:/var/lib/grafana
+      - ./grafana/datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
+      - ./grafana/dashboards.yaml:/etc/grafana/provisioning/dashboards/dashboards.yaml
+      - ./grafana/cloudquery-dashboard.json:/var/lib/grafana/dashboards/cloudquery-dashboard.json
+    environment:
+      GF_FEATURE_TOGGLES_ENABLE: "tempoApmTable"
+    ports:
+      - "3000:3000"
+volumes:
+  prometheus:
+    driver: local
+  grafana_data:
+    driver: local
+  tempo_data:
+    driver: local
+```
+
+This Docker Compose configures [Prometheus][Prometheus], [Tempo][Tempo], an [OpenTelemetry collector][Collector] and Grafana with a custom configuration, and [Loki][Loki] with the default configuration, 
+
+## Step 2: Configure Prometheus
+
+Create a file with the path `prometheus/prometheus.yaml` with the following content:
+
+```yaml
+global:
+  scrape_interval: 15s
+scrape_configs:
+  - job_name: "opentelemetry"
+    static_configs:
+      - targets: ["collector:8090"]
+```
+
+This configuration will tell [Prometheus][Prometheus] to scrape the [OpenTelemetry][OTEL] collector every 15 seconds.
+
+## Step 3: Configure Tempo
+
+Create a file with the path `tempo/tempo.yaml` with the following content:
+
+```yaml
+server:
+  http_listen_port: 3200
+distributor:
+  receivers:
+    otlp:
+      protocols:
+        http:
+storage:
+  trace:
+    backend: local
+    wal:
+      path: /tmp/tempo/wal
+    local:
+      path: /tmp/tempo/blocks
+```
+
+This configuration will tell [Tempo][Tempo] to listen on port 3200 and receive [OpenTelemetry][OTEL] traces via HTTP on the default port of 4318.
```

Comment:
The `[OpenTelemetry][OTEL]` Markdown bit (btw I don't recognise this syntax) is not working in the UI.